src/lib/expression/between_expression.hpp

```diff
@@ -10,17 +10,22 @@ class BetweenExpression : public AbstractPredicateExpression {
  public:
   BetweenExpression(const std::shared_ptr<AbstractExpression>& value,
                     const std::shared_ptr<AbstractExpression>& lower_bound,
-                    const std::shared_ptr<AbstractExpression>& upper_bound);
+                    const std::shared_ptr<AbstractExpression>& upper_bound, bool left_inclusive = true,
+                    bool right_inclusive = true);
```

Comment:
Please update the PR description and especially the title and add the inclusive/exclusive part. When looking at the git log, this would otherwise be missed.

Reply:
@simsieg You are the one, that can update the description?
I think of something like:
To represent single/double side exclusive binary predicate expression the Between Table Scan was extended, too. The BetweenExpression now supports two additional constructor parameters (left_inclusive and right_inclusive), which default to true to comply with the SQL standard.